diff --git a/lib/fog/compute/models/brightbox/account.rb b/lib/fog/compute/models/brightbox/account.rb
index d4b7987..22e1e69 100644
--- a/lib/fog/compute/models/brightbox/account.rb
+++ b/lib/fog/compute/models/brightbox/account.rb
@@ -40,8 +40,8 @@ module Fog
         attribute :telephone_verified
 
         # Times
-        attribute :created_at
-        attribute :verified_at
+        attribute :created_at, :type => :time
+        attribute :verified_at, :type => :time
 
         # Links - to be replaced
         attribute :owner_id, :aliases => "owner", :squash => "id"
diff --git a/lib/fog/compute/models/brightbox/image.rb b/lib/fog/compute/models/brightbox/image.rb
index bda4a80..364f075 100644
--- a/lib/fog/compute/models/brightbox/image.rb
+++ b/lib/fog/compute/models/brightbox/image.rb
@@ -26,7 +26,7 @@ module Fog
         attribute :compatibility_mode
 
         # Times
-        attribute :created_at
+        attribute :created_at, :type => :time
 
         # Links - to be replaced
         attribute :ancestor_id, :aliases => "ancestor", :squash => "id"
diff --git a/lib/fog/compute/models/brightbox/load_balancer.rb b/lib/fog/compute/models/brightbox/load_balancer.rb
index b0de81a..877d999 100644
--- a/lib/fog/compute/models/brightbox/load_balancer.rb
+++ b/lib/fog/compute/models/brightbox/load_balancer.rb
@@ -19,8 +19,8 @@ module Fog
         attribute :listeners
 
         # Times
-        attribute :created_at
-        attribute :deleted_at
+        attribute :created_at, :type => :time
+        attribute :deleted_at, :type => :time
 
         # Links - to be replaced
         attribute :account
diff --git a/lib/fog/compute/models/brightbox/server.rb b/lib/fog/compute/models/brightbox/server.rb
index 235a46c..9f84656 100644
--- a/lib/fog/compute/models/brightbox/server.rb
+++ b/lib/fog/compute/models/brightbox/server.rb
@@ -19,10 +19,10 @@ module Fog
         attribute :console_token
 
         # Times
-        attribute :created_at
-        attribute :started_at
-        attribute :console_token_expires
-        attribute :deleted_at
+        attribute :created_at, :type => :time
+        attribute :started_at, :type => :time
+        attribute :console_token_expires, :type => :time
+        attribute :deleted_at, :type => :time
 
         # Links - to be replaced
         attribute :account_id,  :aliases => "account",      :squash => "id"
