diff --git a/lib/bbcloud/vendor/fog/lib/fog/core.rb b/lib/bbcloud/vendor/fog-0.6.0/lib/fog/core.rb
index dfa61c3..13870f8 100644
--- a/lib/bbcloud/vendor/fog/lib/fog/core.rb
+++ b/lib/bbcloud/vendor/fog/lib/fog/core.rb
@@ -5,7 +5,6 @@ $LOAD_PATH.unshift __LIB_DIR__ unless
   $LOAD_PATH.include?(File.expand_path(__LIB_DIR__))
 
 # external core dependencies
-require 'rubygems'
 require 'base64'
 require 'cgi'
 require 'excon'
